apps/vaporgui/SettingsParams.cpp

```diff
@@ -434,7 +434,8 @@ void SettingsParams::Init()
 {
     SetSessionAutoSaveEnabled(true);
     SetChangesPerAutoSave(5);
-    SetAutoSaveSessionFile("~/VaporAutoSave.vs3");
+    std::string homeDir = QDir::homePath().toStdString();
+    SetAutoSaveSessionFile(homeDir + "/VaporAutoSave.vs3");
```

Comment:
This may be outside the scope of this PR, but I think it is customary to make this a hidden file and provide a menu option to restore it rather than placing a file in the user’s home directory.

Reply:
Agreed, but people are already accustomed to it not being a hidden file. Changing it won't get us much and may cause some confusion.